src/supertux/menu/editor_delete_level_menu.cpp

```diff
@@ -53,12 +53,15 @@ EditorDeleteLevelMenu::menu_action(MenuItem& item)
       Dialog::show_message(_("You cannot delete level that you are editing!"));
     else
     {
-      PHYSFS_delete(m_level_full_paths[id].c_str());
-      delete_item(id + 2);
-      m_level_full_paths.erase(m_level_full_paths.begin() + id);
-      m_level_select_menu->reload_menu();
-      if (!Editor::current()->is_level_loaded())
-        m_levelset_select_menu->reload_menu();
+      Dialog::show_confirmation(_("Are you sure?"), [this, id]()
```

Comment:
Would it be possible to show the name of the level or levelset, that's being deleted, in the message?

Reply:
Yes